packages/lab/src/slider/RangeSlider.tsx

```diff
@@ -23,6 +27,10 @@ export interface RangeSliderProps
    * Disable the slider.
    */
   disabled?: boolean;
+  /**
+   * Show visual ticks above the mark labels
+   */
+  enableMarkTicks?: boolean;
```

Comment:
Perhaps another example where we combining names, marks and ticks, this is creating complexity as the user has to have a mental model of all the parts (or read the docs, which should not be needed)

I think behaviour is normally enabled but in this case, we are changing the presentation, so I would say it should be something like 'showTicks'?